universal-login-relayer/src/integration/sql/services/EmailConfirmationsStore.ts

```diff
@@ -0,0 +1,25 @@
+import Knex from 'knex';
+import {EmailConfirmation} from '@unilogin/commons';
+
+export class EmailConfirmationsStore {
+  private tableName = 'email_confirmations';
+
+  constructor(private database: Knex) {
+  }
+
+  add(emailConfirmation: EmailConfirmation) {
+    return this.database
+      .insert(emailConfirmation)
+      .into(this.tableName)
+      .returning('email');
```

Comment:
`.returning('email')[0]` will return `email` instead of `[email]`

Reply:
<img width="721" alt="Screenshot 2020-07-06 at 15 46 49" src="https://user-images.githubusercontent.com/57775639/86600120-e9b6d480-bf9f-11ea-9e68-da8ab65a7e97.png">

Reply:
You can add `await` before `this.database`

Reply:
<img width="779" alt="Screenshot 2020-07-06 at 16 12 48" src="https://user-images.githubusercontent.com/57775639/86602844-8a5ac380-bfa3-11ea-8770-e45c1410a145.png">

Reply:
Done